### DISCUSSION TITLE
Conservazione crittografata della segnalazione per 5 anni

### DISCUSSION THREAD

**Discussion by marcopvl** [globaleaks/globaleaks-whistleblowing-software#4305] [General]
Buongiorno, 
è possibile sapere se esiste una funzionalità che automatizza la conservazione crittografata per 5 anni della segnalazione e l'eventuale suo recupero per la trasmissione all'autorità ?
Grazie mille

**Comment by evilaliv3** [comment]
Salve @marcopvl 

La legge non impone un tempo minimo di conservazione ma un termine massimo di 5 anni dal termine della gestione.
Ad ogni modo si le policies di conservazione sono tutte configurabili, puoi ad esempio impedire ai riceventi di cancellare i contenuti e impostare una policy di data retention di 5 anni così che ogni contenuto venga mantenuto per 5 anni; ti faccio notare però che questo andrebbe in violazione del GDPR in quanto il GDPR impone di cancellare i dati personali non necessari.

Quando chiedi se sia possibile un eventuale recupero cosa intendi? Se intendi un recupero dopo una cancellazione, no nessun recupero è possibile in alcun modo a seguito di una cancellazione di una seganalzione.
